src/Razor/src/Microsoft.VisualStudio.RazorExtension/RazorPackage.cs

```diff
@@ -72,6 +74,17 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
             var menuToolWin = new MenuCommand(ShowToolWindow, toolwndCommandID);
             mcs.AddCommand(menuToolWin);
         }
+
+        CreateSnippetService();
+    }
+
+    private SnippetService CreateSnippetService()
+    {
+        var componentModel = (IComponentModel)GetGlobalService(typeof(SComponentModel));
+        var joinableTaskContext = componentModel.GetService<JoinableTaskContext>();
+        var cache = componentModel.GetService<SnippetCache>();
+
+        return new SnippetService(joinableTaskContext.Factory, this, cache);
```

Comment:
Assume I know nothing about VSIXes, why does this need to happen in the package? Are we not going to be pinged by RPS for loading in all of the users snippets when there might not even be a razor file ever opened?